synapse/rest/client/v2_alpha/account.py

```diff
@@ -220,12 +220,27 @@ def __init__(self, hs):
         self.auth = hs.get_auth()
         self.auth_handler = hs.get_auth_handler()
         self.datastore = self.hs.get_datastore()
+        self.password_policy_handler = hs.get_password_policy_handler()
         self._set_password_handler = hs.get_set_password_handler()
 
     @interactive_auth_handler
     async def on_POST(self, request):
         body = parse_json_object_from_request(request)
 
+        # we do basic sanity checks here because the auth layer will store these
+        # in sessions. Pull out the username/password provided to us.
```

Comment:
this comment doesn't seem to match reality. in particular, we're not pulling out the username here?

Reply:
Oops, this is what I get for copying and pasting. Thank you! 👍